internal/core/legacy/services/permission/permission.go

```diff
@@ -172,6 +173,11 @@ var checkAdaptor security.PermissionAdaptor
 func (p *Permission) CheckPermission(req *apistructs.PermissionCheckRequest) (bool, error) {
 	logrus.Debugf("invoke permission, time: %s, req: %+v", time.Now().Format(time.RFC3339), req)
 
+	// if the param is invalid, it will set the req as nil
+	if req == nil {
+		return false, errors.New("the user operation is invalid, maybe the param is invalid")
```

Comment:
make it concise: `empty request`

Reply:
modified the param validation logic of orgID, and the issue has been fixed